person.rb

```diff
@@ -1,4 +1,5 @@
-class Person
+require '.\nameable'
```

Comment:
- [ ] Please kindly check my detailed comments on the `trimmer_decorator` file to effect changes suggested by the last reviewer here which I explained very well. 👍